### ISSUE TITLE
[20220116] Weekly AI ArXiv 만담 - 36회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#36]
- News
  - Conference 소식
    - ACL commitment + rebuttal + NAACL 제출 수고 많으셨습니다. 
    - ICML 2022 abs deadline 은 20일 입니다. 
  - NAVER Webtoon AI Video Webtoon!: https://webtoon.github.io/WebtoonMe/
  - https://user-images.githubusercontent.com/11782739/149538436-c2410e54-9082-406d-b58e-e4a10a2ba96b.mp4
- ArXiv
  - [A ConvNet for the 2020s](https://arxiv.org/abs/2201.03545) by 이진원님
  - [Detecting Twenty-thousand Classes using Image-level Supervision](https://arxiv.org/abs/2201.02605)
    - 기존 Detector보다 훨씬 많은 object vocabulary를 커버하는 신개념 detector from Meta AI
    - Detection 데이터도 image-level label 데이터도 둘다 학습 가능 (Open vocabulary detector 위해 CLIP embedding 활용)
    - GT label + box 개념이 아니라 GT label을 포함하는 전체 큰 박스를 활용
    - LVIS, Openvoc COCO, Openimages, Object365 등 평가해서 상당한 개선
    - https://github.com/facebookresearch/Detic
![image](https://user-images.githubusercontent.com/11782739/149658977-181f4ade-ba72-4383-855b-7cddf76d22ca.png)
![image](https://user-images.githubusercontent.com/11782739/149658994-f473867a-c06c-41e2-8b31-1e5fcf51eb9c.png)
![image](https://user-images.githubusercontent.com/11782739/149659127-80692381-6aeb-450f-ade9-c7d2255ee1c0.png)
   
  - [Contrastive Fine-grained Class Clustering via Generative Adversarial Networks](https://arxiv.org/abs/2112.14971)
    - GAN 기반의 Fine-graind image class clustering 기법
    - 기존 연구는 box annotation supervision이 필요했고 없으면 성능이 별로 였음.
    - 배경, 물체 분리해서 cluster 와 latent sampling으로 컨트롤 해서 다양한 이미지가 생성되면서 미세한 detail feature 학습되도록 generator 와 discriminator 학습
    - 입력되는 이미지는 D를 통과해서 나오는 feature를 가장 가까운 cluster centroid에 매핑
    - 그래서 클러스터링도 되면서 이미지 생성도 잘되는 모델
    - https://github.com/naver-ai/c3-gan
![image](https://user-images.githubusercontent.com/11782739/149659314-37b2221f-3edf-4cb8-a509-319f35c48045.png)
![image](https://user-images.githubusercontent.com/11782739/149659334-5da6f7b2-f28e-43cc-ac05-f3d6b7205610.png)
![image](https://user-images.githubusercontent.com/11782739/149659355-8d575a33-4d57-4611-9d48-5c96cd3bf47a.png)

**Comment by ghlee0304** [comment]
- Arxiv
    - [Music2Video: Automatic Generation of Music Video with fusion of audio and text](https://arxiv.org/abs/2201.03809)
    - [MR-SVS: Singing Voice Synthesis with Multi-Reference Encoder](https://arxiv.org/abs/2201.038641)

**Comment by jwlee-ml** [comment]
- [A ConvNet for the 2020s](https://arxiv.org/abs/2201.03545)
     - ViT, Swin Transformer와 같은 computer vision 분야의 transformer 계열 network에 대한 CNN의 반격(?)
     - ConvNet의 현대화라는 이름으로 여러가지 테크닉들을 적용하여 ConvNeXt라는 새로운 network을 제안
     - ViT, Swin, RegNet, EffNet 등등보다 classification, detection, segmentation에서 모두 동등 혹은 우월한 성능을 보여주었음
     - ResNet에 새롭게 적용한 기술들
         - modern training techniques: AdamW, longer training, various augmentations, stochastic depth, label smoothing, etc.
         - changing stage compute ratio: (3, 4, 6, 3) -> (3, 3, 9, 3)
         - patchifying stem network: 4x4 conv w/ stride 4
         - ResNextify: using depthwise convolution(group conv, # of groups = # of channels)
         - inverted bottleneck: like mobilenet v2
         - moving up detphwise conv layer: depthwise conv위치를 1x1 위로 올림(먼저 수행)
         - increasing the kernel size: 3x3 -> 7x7
         - ReLU -> GELU
         - fewer activation functions: single GELU activation in each block
         - fewer normalization layers, layer norm: single layer norm layer in each block
         - separate downsampling layers
         
![image](https://user-images.githubusercontent.com/25281219/149655829-0b0be77e-778d-46bf-b830-fdd7decfa9f9.png)
![image](https://user-images.githubusercontent.com/25281219/149655995-1af6eced-7777-4992-bc38-a08d13b7ad11.png)
![image](https://user-images.githubusercontent.com/25281219/149655813-69326d7d-4d7e-4d08-8093-9d54776e1db8.png)
![image](https://user-images.githubusercontent.com/25281219/149656033-33482ee4-5703-42ff-831c-23c3115bdb4a.png)

**Comment by kimyoungdo0122** [comment]
- Arxiv
    - [AI that understands speech by looking as well as hearing](https://ai.facebook.com/blog/ai-that-understands-speech-by-looking-as-well-as-hearing/)
        - 사람이 기계보다 대화를 더 잘 인식할 수 있는 이유는 귀로만 듣는 것이 아니라 눈으로 상대방을 보고 있기 때문
        - 그래서 음성 인식 + 화자 입술 인식을 더해서 Audio-Visual Speech Recognition을 실시
        - self supervised learning을 활용해서 레이블된 데이터가 적을 때에도 꽤 잘 작동한다고 합니다
        - [Robust Self-Supervised Audio-Visual Speech Recognition](https://arxiv.org/abs/2201.01763?fbclid=IwAR1xU_wYAGP5_4vQHwEdiMRbYB30YV8RJy7WkKz5CQ1fMX6VdHNhjE0SMTo)
        - [https://arxiv.org/abs/2201.02184?fbclid=IwAR3kJaVQyhd-_wAWPjuGyZCohW-4zEgIJ_dI8LL7cLsYGvGcbwra4vzDZ54](https://arxiv.org/abs/2201.02184?fbclid=IwAR3kJaVQyhd-_wAWPjuGyZCohW-4zEgIJ_dI8LL7cLsYGvGcbwra4vzDZ54)
![image](https://user-images.githubusercontent.com/42539153/149655797-dbf307e5-347f-40bd-9d29-ebfb944d974e.png)
![image](https://user-images.githubusercontent.com/42539153/149656030-67ff55ff-6d9c-491d-aeaa-eb95bb88e23a.png)
        - 향후 Multilingual으로도 연구할 예정이고, 언어 장애가 있는 사람을 위한 도구로도 잘 작동할 것으로 전망
        
- News
    - [Chinese scientists develop AI ‘prosecutor’ that can press its own charges](https://www.scmp.com/news/china/science/article/3160997/chinese-scientists-develop-ai-prosecutor-can-press-its-own)
        - 중국에서 기소까지 가능한 AI '검사'를 개발
        - 사건에 대한 구두 설명 데이터로 97% 정확도를 기록
        - 15년부터 20년까지 총 17,000건 이상의 사건 사례를 학습
        - credit card fraud, running a gambling operation, dangerous driving, intentional injury, obstructing official duties, theft, fraud (신용카드 사기, 도박판 운영, 난폭 운전, 고의적인 상해, 공무집행 방해, 절도, 사기) 등을 구분할 수 있음
    -

**Comment by visualxlab** [comment]
* Arxiv
  * [face analysis in the wild using synthetic data alone](https://arxiv.org/pdf/2109.15102.pdf)
  * 3D 얼굴 모델(obj)을 사용해, 얼굴 합성에 필요한 데이터 라벨링을 수작업으로는 불가능한 영역까지 확장

![image](https://user-images.githubusercontent.com/80636583/149658908-31a3541f-7ab2-4d65-9ed9-5b5d1b25081a.png)
![image](https://user-images.githubusercontent.com/80636583/149659427-34e5b851-1d6b-4556-9495-5cf49f93183d.png)
![image](https://user-images.githubusercontent.com/80636583/149659456-bd183d97-8bfd-415d-878b-f86097aa0e24.png)

**Comment by veritas9872** [comment]
**Structure and position-aware graph neural network for airway labeling**
Arxiv: https://arxiv.org/abs/2201.04532
GitHub: https://github.com/DIAGNijmegen/spgnn (아직 없음).

흉부 의료영상에서 저명하신 네덜란드의 Bram van Ginneken 교수님의 연구를 공우해드립니다.

Segmentation이 되어 있는 Lung CT 영상에서 각 airway가 구체적으로 폐의 어떤 airway인지 classification하는 연구에 CNN과 Graph Transformer를 사용한 연구입니다.

Airway의 구조는 큰 가지와 작은 가지의 구조에서 큰 차이가 존재하는데 작은 가지는 개인마다 차이가 크지만 대체로 큰 가지를 비슷합니다. 이때 해당 가지가 어떤 segment인지 분류하는 것을 하기 위해 기존에는 순수 CNN classification 방식이나 segmentation 방식을 적용했었으나 본 논문에서는 구조 정보와 위치 정보를 혼합한 구조를 사용합니다.

조금 아쉬운 점은 segmentation map이 이미 제공되어야 한다는 점인 것 같습니다.

![Screenshot (10)](https://user-images.githubusercontent.com/33523965/149660779-d446cfa4-1764-46b0-818c-b60fcda56e25.png)


![Screenshot (11)](https://user-images.githubusercontent.com/33523965/149660771-0b2a6666-0a4a-4090-b001-0e4f85ba1ac7.png)

**Comment by hollobit** [comment]
### 선관위, 딥페이크 영상 기반 선거운동에 대한 법규 해석 발표 
https://www.nec.go.kr/site/vt/ex/bbs/View.do?cbIdx=1231&bcIdx=156972&relCbIdx=1084
- 딥페이크 영상을 이용해 선거운동을 할 수는 있음. 그러나 딥페이크영상(AI○○○)임을표시하지않는경우「공직  선거법」(이하‘법’이라함)제250조(허위사실공표죄)에위반됨
- 방송연설에서의 딥페이크 영상 활용은 원천 금지. 선관위는 “방송연설의 주체는 후보자 본인이기 때문에 후보자를 대신해 딥페이크 영상을 방영할 수는 없다"고

**Comment by terryum** [comment]
- ["Most Time Series Anomaly Detection Papers are Wrong" (Reddit)](https://www.reddit.com/r/MachineLearning/comments/rwwjal/d_a_paper_suggests_most_time_series_anomaly/)

어떤 분이 흥미로운 논문을 발견(?)해서 1월 5일 Reddit에 공유를 했네요. Time-series anomaly detection (TAD) 문제의 evaluation 과정에서 후처리로 쓰는 point adjustment (PA)라는 프로세스가 있는데, PA만 있으면 random prediction도 SOTA를 찍는 놀라운 변신 능력(?)을 보여준다고 합니다. 즉, 성능평가에 있어 PA를 적용한 여러 TAD 연구들의 성능이 매우 과장되어 있다는 것이죠.

원 논문은 (흥미롭게도) 한국에서 나온 논문입니다. (will appear in AAAI-22)
[Towards a Rigorous Evaluation of Time-series Anomaly Detection](https://arxiv.org/abs/2109.05257)
Siwon Kim, Kukjin Choi, Hyun-Soo Choi, Byunghan Lee, Sungroh Yoon

PA는 "ground truth 입장에서 보았을 때 하나라도 point-wise prediction에 걸렸다면 모두를 맞춘 것으로 친다"라고 하는 '하나만 걸려라' 식 후처리입니다. 아래 그래프에서처럼 딱 하나만 point prediction이 threshold를 넘어도 GT를 참고해 모든 point를 맞춘것처럼 prediction을 수정해 주는거죠.
![image](https://user-images.githubusercontent.com/12528769/149661520-3660f285-8617-4f27-96fd-f5ea4bed4ee5.png)
![image](https://user-images.githubusercontent.com/12528769/149661640-4f194acf-3461-473c-8c85-223a3fc74fac.png)

우습게도(?) PA를 적용하면 현존하는 최고의 TAD 알고리즘은 random prediction이 된다고 하네요.
![image](https://user-images.githubusercontent.com/12528769/149661757-57dcafda-0059-4e37-8938-e42435b2b422.png)

Reddit의 댓글에는 time-series prediction에서 꾸준히 재기되어왔던 성능평가에 대한 이슈를 비판하며 "오늘의 날씨 = 어제의 날씨"와 같은 냉소적인 댓글이 달리고 있네요.


[Terry의 의견]
학계에서 말하는 '벤치마크에서 높은 성능을 찍었다'라는 것과 실제 현장에서 '이 알고리즘이 정말 쓸만하다'라는 것과 차이가 날 때가 많습니다. 때로는 데이터의 가정이 현실적이지 않아서, 때로는 현실에선 논문처럼 문제가 명확하게 정의되지 않아서, 때로는 evaluation이 현실문제와는 동떨어져서 등의 이유로 현실에선 못쓰이는 경우가 많죠. 하지만 여전히 많은 연구자들은 "쓸모없는 문제"에서 SOTA를 찍으려 노력합니다. 

현실에서 가장 어려운 문제 중 하나가 '이 문제를 잘 풀었다는 것을 어떻게 evaluation 하는가?'에 대한 답을 내는 것입니다. 현실문제를 수학적으로 모델링하고, 그 성능을 하나의 숫자로 표현 한다는 건 정말 어려운 일이죠 (그것만으로도 AI에서 큰 업적을 냈다고 할만합니다.) 하지만 TAD 문제가 어렵단 이유로 이렇게 관대한 metric을 2018년에 제안하고 아직까지도 써왔다는 것에 대해선 유감입니다. 아마도 논문의 우수성을 증명하고 싶었겠지만, 어려운 문제는 과장없이 '어렵다'는 결론으로 두는게 맞지 않을까 싶습니다.


[Reference] PA를 처음 evaluation에 사용한 논문:
[Unsupervised Anomaly Detection via Variational Auto-Encoder for Seasonal KPIs in Web Applications, H. Su et al. ](https://arxiv.org/abs/1802.03903)
